chapter05/chapter05.rpy

```diff
@@ -137,7 +137,7 @@ label c05_intro:
     I stared at the now-embarrassed Celia, trying to figure out her motives.
     Under my gaze, Celia's eyes wander the room, a soft blush forming on her cheeks.
     ... [Pause] Cute.
-    Looking down at the sketchbook, I carefully ripped a single sheet of paper along with the holed-rom, making the paper clean.
```

Comment:
Aha, they are indeed a thing. I'm not much of an artist so my experience is limited haha. Thanks, I'll rework the sentence.